### DISCUSSION TITLE
Is there a way to bypass LVAR?

### DISCUSSION THREAD

**Discussion by MStallen** [flybywiresim/aircraft#2922] [Q&A] [answered]
I think we've to wait for a long time to have access to LVAR's. I was wondering if it is possible to write some actions to non occupied simvar places, so you can use them via FSUIPC. For instance the EFIS actions like ND mode and zoom, terrain-button etc.

**Comment by Benjozork** [comment, marked as answer]
Hello,

Doing so would make code extremely confusing and would potentially cause lots of undefined behavior.

There is a way to access LVars if you create an HTTP/WebSocket client that communicates with a server on your machine, realying necessary information (since JavaScript can access LVars).

This idea has been considered but for both of the above reasons, was decided against.
